src/dotnet/commands/dotnet-add/dotnet-add-package/Program.cs

```diff
@@ -94,7 +100,7 @@ public override int Run(string fileOrDirectory)
                 GetProjectDependencyGraph(projectFilePath, tempDgFilePath);
             }
 
-            var result = NuGetCommand.Run(TransformArgs(RemainingArguments.First(), tempDgFilePath, projectFilePath));
+            var result = NuGetCommand.Run(TransformArgs(_packageNameArgument.Value, tempDgFilePath, projectFilePath));
```

Comment:
Please ensure one of the pack tests passes the option/argument in a previously non-working order so we have validation of the issue we're actually fixing with this PR.